soc/arm/microchip_mec/mec172x/Kconfig.defconfig.series

```diff
@@ -16,6 +16,15 @@ config NUM_IRQS
 
 source "soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172x*"
 
+config FLASH_SIZE
+	default $(dt_node_reg_size_int,/flash@c0000,0)
```

Comment:
Document this is MEC1723 EC ROM bootloader specific to prevent attempts in customization

Reply:
it's SoC-specific, not ROM bootloader specific, but yes needs documented.